tokens/src/core/global/typography.json

```
@@ -22,7 +23,7 @@
         },
         "monospace": {
           "source": "$font-family-monospace",
-          "$value": "SFMono-Regular, Menlo, Monaco, Consolas, 'Liberation Mono',  'Courier New', monospace",
+          "$value": ["SFMono-Regular, Menlo, Monaco, Consolas, 'Liberation Mono',  'Courier New', monospace"],
```

It looks like this `$value` is an array with a single item vs. an array containing items for each font-family. Should it be something like the following?

```suggestion
          "$value": ["SFMono-Regular", "Menlo", "Monaco", "Consolas", "Liberation Mono",  "Courier New", "monospace"],
```

---

That matches up with the syntax used in https://github.com/openedx/paragon/pull/3222#discussion_r1803396628

---

Corrected, thanks!